### PR TITLE
PandExo 2.0 Ready To Merge

### PR DESCRIPTION
**Pull request by natashabatalha**
Merge to incorporate new Pandeia.

The figure shows expected differences in group number calculations and precision updates. This is a major update that should be done before submitting Cycle 2 JWST proposals. 

Screen Shot 2022-12-08 at 5 48 01 PM

Major installation changes: 

- Updated pandeia requires the download of throughput files. These new instructions have been added to the installation docs.

### CODE REVIEW

**Comment by nespinoza** [comment]
Testing in the online version happening through the efforts of @mfixstsci. Will report when testing is done!

**Review by nespinoza — APPROVED** [review]
All right @natashabatalha, sorry about the delay on this. I've personally installed the newest version of PandExo and all goes smoothly.

@mfixstsci performed a heroic effort to build the online version as well and all works nicely. I tested all the modes with a couple of examples and runs without problems except for a few items, below. Approving contingent on those being looked I guess?

The three small issues we found were:

- PandExo fails when setting the number of groups manually, at least on the online version. If I set that to, say, 20, I get the little falling Panda with an error:

        "/myfolder/PandExo/pandexo/engine/jwst.py", line 78, in compute_full_sim
            if 'optimize' in conf['detector']['ngroup']:
        TypeError: argument of type 'int' is not iterable
        
    Found it interesting that this didn't happen before; perhaps this was allowable with Python 2? At least with Python > 3.8, it's not the case anymore.

- For MIRI runs, the automatically-installed version of Astropy has hard coded the standard spectrum of vega to an old version from [calstis](https://archive.stsci.edu/hlsps/reference-atlases/cdbs/current_calspec/). The one Astropy wanted is `alpha_lyr_stis_010.fits` but the current one (and the one you get when `calspec` is automatically downloaded is `alpha_lyr_stis_011.fits`. I'm unsure if this will be a problem accross platforms or if this was just an issue with Python 3.8 (the one we used for testing), so probably not a dealbreaker (and not PandExo's fault).

- Related to the above: README and docs claim Python > 2.7 is needed. This version of PandExo can only do >= 3.8.

Bottom line: mostly all good from our tests over here!

Thanks for making this update!
Néstor
